content/yaml-code-signing/signing-macos.md

```diff
@@ -131,6 +131,10 @@ Instead of specifying the exact bundle-id, you can use `"$(xcode-project detect-
 
 ## Provide signing files manually
 
+{{<notebox>}}
+If you are using [Codemagic Teams](../teams/teams), then signing files, such as certificates and provisioning profiles, can be managed under the [Code signing identities](./code-signing-identities) section in the team settings and do not have to be uploaded as environment variables as in the below instructions. However, note that some functionality may be limited.
+{{</notebox>}}
+
```

Comment:
I'm not sure we should include this info here as we currently focus on signing iOS apps in code signing identities

Reply:
Well, the profiles and certs can be uploaded regardless, I think it should b here.